itools-packager/src/main/java/com/powsybl/itools/ItoolsPackagerMojo.java

```diff
@@ -181,7 +217,13 @@ public void execute() {
 
             getLog().info("Zip package");
```

Comment:
move this log under the `zip` case, add a new `getLog().info("Tgz package");` for the tgz case